draft-dekater-scion-dataplane.md

```diff
@@ -1374,6 +1374,11 @@ In addition, the MAC algorithm can be freely chosen by each AS, enabling algorit
 A more realistic risk to the secrecy of the forwarding key is exfiltration from a compromised router or control plane service.
 An AS can optionally rotate its forwarding key at regular intervals to limit the exposure after a temporary device compromise. However, as is perhaps self-evident, such a key rotation scheme cannot mitigate the impact of an undiscovered, permanent compromise of a device.
 
+When an AS's forwarding key is compromised, an attacker can forge hop field MACs, undermining path authorization. Recall that path segments are only checked for validity and policy compliance during the path discovery phase, and routers only perform minimal sanity checks of hop fields with valid MACs. Consequently, creating fraudulent hop fields with valid MACs allows an attacker to bypass most path segment validity checks, and to create path segments that violate the AS's local policy and/or general path segment validity requirements.
+In particular, an attacker with access to the forwarding key of at least two compromised ASes could create paths that include loops (limited by the maximum number of hop fields of a path).
+
+While a forwarding key compromise and the resulting loss of path authorization is a serious degradation of SCION's routing security properties, this does not affect, for example, access control or data security for the hosts in the affected AS.
```

Comment:
Added another sentence to clarify that this does not affect routing for legitimate senders.

True, the attacker will most likely try to use this to splice parts of path segments, but it turns out this is quite involved because of the MAC chaining. I tried to explain this as short as possible.